tests/console/force_cron_run.pm

```diff
@@ -14,24 +14,31 @@
 use base "consoletest";
 use strict;
 use testapi;
-use utils 'assert_screen_with_soft_timeout';
+use utils qw(assert_screen_with_soft_timeout is_jeos);
+
+sub check_load {
+    script_run "top; echo TOP-DONE-\$? > /dev/$serialdev", 0;
+    # Let the load settle.
+    # On JeOS we begin with higher load so it takes longer for the load to settle.
```

Comment:
If you could put coolo's explanation for JeOS here then I think it would be ok to handle it different than default SLE